resources/zib/zib-BodyHeight.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-BodyHeight" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-BodyHeight" />
+  <name value="ZibBodyHeight" />
+  <title value="zib BodyHeight" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A person’s body height." />
```

Comment:
https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4#Metadata

Reply:
Description changes to "Body weight is the common name for (human) body mass."

Reply:
Wow, sorry, dit is mijn fout. Het stond goed.